starwars/src/App.js

```diff
@@ -26,11 +29,22 @@ class App extends Component {
       })
       .then(data => {
         let thedata = data;
+        console.log(thedata);
         // this.setState({starwarsChars: data.results});
         this.setState(prevState => {
           const data = thedata.results;
           return {starwarsChars: data};
         });
+
+        this.setState(prevState => {
```

Comment:
`const dNext = thedata.next; 
this.setState({ dataNext:  dNext});`
Here is a different way of doing it. You don't need prevState in this situation. It is only used when you need the prevState to be altered like it will toggle if it true or false based on it's previous state.